lib/screens/login/login_keys.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/material.dart';
+
+class LoginKey {
+  static const liLoginEmail = Key('liLoginEmail');
```

Comment:
What do these words: `li` & `rrb` mean? 🤔

Reply:
@lydiasama We don't have this convention, yet IMO. Can you raise it to the flutter-guild, please? 🙏